### DISCUSSION TITLE
setValue, shouldTouch one or several fields callback

### DISCUSSION THREAD

**Discussion by ganijon-dev** [react-hook-form/react-hook-form#3731] [Ideas]
We are building forms and we have several needs which I think many people will need too. 
setValue currently sets only in one field but in case if you have dropdown with many information on it onSelect event I want to setValue in 2-or more fields due to the informations already stored in dropdown and of course shouldTouched similar to shouldDirty , shouldValidate  would be great option since the field is already filled in order to puck Checked Icon or so what
https://www.loom.com/share/be13f6e8ea184bb2b56fce9588b14ddd

**Comment by bluebill1049** [comment]
please send a proper feature request, we will put RFC against this and have a proposal to build this feature if demand is high.
